sktime/forecasting/tests/test_sktime_forecasters.py

```diff
@@ -42,6 +42,20 @@
     if issubclass(forecaster, _BaseWindowForecaster)
 ]
 
+
+# helper function
+def _get_n_columns(tag):
```

Comment:
Let's move this to `https://github.com/alan-turing-institute/sktime/blob/main/sktime/utils/_testing/forecasting.py` and re-use it in both files.